src/App.vue

```diff
@@ -125,34 +131,40 @@
 
 <script lang="ts">
 // Libraries
-import { Component, Vue, Watch, Mixins } from 'vue-property-decorator'
+import { Component, Mixins, Vue, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'vuex-class'
 import KeycloakService from 'sbc-common-components/src/services/keycloak.services'
 import { PAYMENT_REQUIRED } from 'http-status-codes'
 import * as Sentry from '@sentry/browser'
-import { updateLdUser } from '@/utils'
+import { updateLdUser, getFeatureFlag } from '@/utils'
 
 // Components
 import PaySystemAlert from 'sbc-common-components/src/components/PaySystemAlert.vue'
 import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
 import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
 import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
-import { EntityInfo, Stepper, Actions } from '@/components/common'
-import Views from '@/views'
+import { Actions, EntityInfo, Stepper } from '@/components/common'
+import * as Views from '@/views'
```

Comment:
Why?

Reply:
Because we now have multiple directories in Views, with the root of `views` exporting multiple directories from index. 
We need to assign these multiple exports a reference in App.vue before we can `spread` it, this is what worked at least!

Reply:
Ah, it's because you're (re)exporting 3 things in src/views/index.ts instead of just one thing as before.

Another way to do this is to import all 3 things and then export them as one.

Reply:
Cool, will look at it!